packages/augur-core/source/contracts/WarpSync.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity 0.5.10;
+
+import 'ROOT/IWarpSync.sol';
+import 'ROOT/reporting/IUniverse.sol';
+import 'ROOT/reporting/IMarket.sol';
+import 'ROOT/reporting/IV2ReputationToken.sol';
+import 'ROOT/libraries/Initializable.sol';
+
+
+contract WarpSync is IWarpSync, Initializable {
+
+    struct Data {
+        uint256 warpSyncHash;
+        uint256 timestamp;
+    }
+
+    IAugur public augur;
+    mapping(address => address) public markets;
+    mapping(address => Data) public data;
+
+    uint256 private constant MARKET_LENGTH = 1 days;
+    uint256 private constant MAX_NUM_TICKS = 2 ** 256 - 2;
+    int256[] private PRICES = [0, 1 ether];
+    string private constant EXTRA_INFO = '{"description":"What will the next Augur Warp Sync hash be?","longDescription":"What will the Augur SDK warp sync hash be for the latest block time before or equal to the end date of this market?"}';
```

Comment:
Two things:
1. Could we go this with a blocknumber vs a time for the hash?
2. Should it be the hash at end of the *last* market?

Reply:
1. The problem with blockNumber is that it makes wording the market more difficult and potentially could in theory result in an invalid market if the corresponding block number hasn't been mined.
2. The stored hash is for the last resolved market which will give us the latest hash avaialble.